tutorials/tmva/TMVA_CNN_Classification.C

```diff
@@ -461,6 +461,21 @@ void TMVA_CNN_Classification(std::vector<bool> opt = {1, 1, 1, 1})
       }
    }
 
+   if (usePyTorchCNN) {
```

Comment:
the variable usePyTorchCNN is not defined. 
Please add this line at the beginning of the function TMVA_CNN_Classification:
```
 bool usePyTorchCNN = (opt.size() > 4) ? opt[4] : false; 
````